Winapp2.ini

```diff
@@ -1257,9 +1301,9 @@ FileKey18=%LocalAppData%\Torch\User Data\*|WebRTCIdentityStore
 FileKey19=%LocalAppData%\Vivaldi\User Data\*|WebRTCIdentityStore
 FileKey20=%LocalAppData%\Yandex\YandexBrowser\User Data\*|WebRTCIdentityStore
 
-; End of Chrome/Chromium based browsers.
-;
-; Firefox/Mozilla based browsers.
+; End of Chrome/Chromium based browsers
+; 
+; Firefox/Mozilla based browsers
 

```

Comment:
Are the changes to the comment line intended? If so, the dot at the end of the start line should also be removed:

`; Chrome/Chromium based browsers`